pkg/converter/convert_unix.go

```diff
@@ -919,7 +919,9 @@ func convertManifest(ctx context.Context, cs content.Store, oldDesc ocispec.Desc
 		return nil, errors.Wrap(err, "read manifest json")
 	}
 
-	if isNydusImage(&manifest) {
+	// If opt.EncryptRecipients is empty, the Nydus image does not need to be
+	// encrypted after conversion.
+	if len(opt.EncryptRecipients) == 0 && isNydusImage(&manifest) {
```

Comment:
Does this work if the user needs encryption on the first conversion but not on the second?

Reply:
Every time `convert` is called seperatly, the corresponding opt will be generated according to the configuration. If there are multiple calls to `convert` in the life cycle of one opt, they will be handled the same. Do we have this scenario?

Reply:
If users want to do conversion with different encryption behaviors, they have to modify the configration file or the cmd args.

Reply:
> Every time `convert` is called seperatly, the corresponding opt will be generated according to the configuration. If there are multiple calls to `convert` in the life cycle of one opt, they will be handled the same. Do we have this scenario?

Multiple calls to convert are possible, such as using acceld/nerdctl (imported the Converter package) to convert an image from containerd's content store.

Reply:
> 



> > Every time `convert` is called seperatly, the corresponding opt will be generated according to the configuration. If there are multiple calls to `convert` in the life cycle of one opt, they will be handled the same. Do we have this scenario?
> 
> Multiple calls to convert are possible, such as using acceld/nerdctl (imported the Converter package) to convert an image from containerd's content store.

User may need to modify the conguration file or the cmd arg according to his needs.